domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -23,6 +26,13 @@ public CQLWrapper() {
     super();
   }
```

Comment:
Can you remove this redundant constructor?

Reply:
Sure

Reply:
`super()` has been removed, but the constructor is still there.